src/emulator/modules/SceLibKernel/SceLibKernel.cpp

```diff
@@ -357,26 +351,26 @@ EXPORT(int, sceIoDevctlAsync) {
 }
 
 EXPORT(int, sceIoDopen, const char *dir) {
-    return open_dir(host.io, dir, host.pref_path.c_str(), export_name);
+    return open_dir(host.io, dir, host.pref_path, export_name);
 }
 
-EXPORT(int, sceIoDread, SceUID fd, emu::SceIoDirent *dir) {
```

Comment:
I think `emu::` is correct here.

Reply:
I removed the `emu::` here in favor of `vita-headers`' definition of SceIoDirent.

Reply:
I don't think you should. The only difference between the `emu::` struct and vita-header's struct is that the pointer `d_private` is of correct size (4 bytes instead of 8).

Reply:
Confirmed that up on Discord (wanted a second opinion), so I reverted the changes.